src/js/test/initTestStore.js

```diff
@@ -23,7 +23,9 @@ export default (zealot: * = createZealotMock()): TestStore => {
     undefined,
     compose(
       applyDispatchAll(),
-      applyMiddleware(reduxThunk.withExtraArgument({zealot})),
+      applyMiddleware(
+        reduxThunk.withExtraArgument({zealot, createZealot: createZealotMock})
```

Comment:
For this I was thinking....

```js
function initTestStore(zealot = createZealotMock()) {
  //...
  const createZealot = () => zealot
  //...
  reduxThunk.withExtraArgument({createZealot})
  //...
}
```

This way we could keep the method signature of `iniTestStore` the same in all the tests, and continue to stub responses in the same way we have been doing.